src/components/sidebar/index.tsx

```diff
@@ -100,20 +107,37 @@ const Sidebar = ({
 	}
 
 	let sidebarContent
+	const filteredMenuItems = getFilteredNavItems(itemsWithMetadata, filterValue)
+	const navResourceItems = generateResourcesNavItems(currentProduct?.slug).map(
+		(item) => item
+	)
 	if (children) {
 		sidebarContent = children
-	} else {
-		const filteredMenuItems = getFilteredNavItems(
-			itemsWithMetadata,
-			filterValue
+	} else if (isInstallPage) {
+		sidebarContent = (
+			<OpenApiSidebarContents
+				navItems={filteredMenuItems as unknown as OpenApiNavItem[]}
+				navResourceItems={navResourceItems.splice(1)}
+				showFilterInput={false}
+			/>
 		)
+	} else {
 		sidebarContent = (
-			<SidebarNavList>
-				{filteredMenuItems.map((item: FilteredNavItem, i) => {
-					const key = `${item.id}-${i}`
-					return <SidebarNavMenuItem item={item} key={key} />
-				})}
-			</SidebarNavList>
+			<>
+				<SidebarNavList>
+					{filteredMenuItems.map((item: FilteredNavItem, i) => {
+						const key = `${item.id}-${i}`
+						return <SidebarNavMenuItem item={item} key={key} />
+					})}
+				</SidebarNavList>
+				<SidebarHorizontalRule />
+				<SidebarNavList>
+					{navResourceItems.map((item, index) => (
+						// eslint-disable-next-line react/no-array-index-key
+						<SidebarNavMenuItem item={item} key={index} />
+					))}
+				</SidebarNavList>
```

Comment:
Had to move resources nav here because `navResourceItems` is also passed to `			<OpenApiSidebarContents>`. If it remained where it was, there would be two instances of the Resources nav on in the install page. 

<img width="769" alt="Screenshot 2023-11-29 at 7 36 38 PM" src="https://github.com/hashicorp/dev-portal/assets/55773810/9a8b3596-7b17-48b5-8337-729493371989">